packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -33,6 +33,60 @@ import {
   XvdSparkLines,
 } from 'xo-sparklines'
 
+const GuestToolsDetection = ({ vm }) => {
```

Comment:
According to [this comment in the issue](https://github.com/vatesfr/xen-orchestra/issues/4783#issuecomment-626594562), it also depends on virtualization mode.

Reply:
See https://github.com/vatesfr/xen-orchestra/pull/5007/files#diff-822ef9925d865be023259e6060ce809cR45